openslides/agenda/config_variables.py

```diff
@@ -23,6 +23,16 @@ def get_config_variables():
         subgroup="General",
     )
 
+    yield ConfigVariable(
```

Comment:
Two options:
1) leave this config in the agenda, but make it unspecific -> change the label and do not meantion submitters
2) move this to motions and change the label to `Show a subtitle with submitters in the agenda`.

If option  2 is choosen, the motion repository must inject the value for this config into the `ViewMotion`'s `getAgendaSubtitle`.